content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Pod Eviction
```

Comment:
```suggestion
title: Eviction
```
You can't evict (or schedule, or preempt, or deschedule) anything other than a pod.